Surface_mesh_topology/doc/Surface_mesh_topology/Surface_mesh_topology.txt

```diff
@@ -262,9 +285,51 @@ The canonical form of a curve is obtained by flattening its brackets, removing i
 \subsection SMTopology_Homotopy_Test Homotopy Test
 It can be proven that the canonical form is uniquely defined and only depends on the homotopy class of the curve. Hence, the curves \f$C'\f$ and \f$D'\f$ in \f$\cal{Q}\f$ are homotopic if and only if their canonical forms are equal. Since each curve is defined as a sequence of (oriented) edges up to a cyclic permutation, we resort to the Knuth-Morris-Pratt algorithm to decide in linear time if the canonical forms are the same up to a cyclic permutation.
 
+\subsection SMTopology_Simplicity_Test Simplicity Test
+It can be shown that a closed curve is homotopic to a simple one if and only if its canonical form can be made intersection free via infinitesimal perturbations and some homotopy preserving operations. One can imagine each edge in the quadrangulation has width and each vertex in the quadrangulation has area so that paths visiting the same vertex/edge multiple times can avoid intersection within a vertex or an edge. See \cgalFigureRef{fig_perturbation_sample} for an example.
+\cgalFigureBegin{fig_perturbation_sample,perturbation_sample.svg}
+ Applying perturbation to reduce 2 intersections between red subpath and blue subpath.
+    \cgalFigureEnd
+
+Such perturbation can be expressed as orderings of edges from the canonical form which cross the same edge in the quadrangulation. The idea of the algorithm is to inductively build such ordering and try to avoid intersection as best as we can.
+
+\subsubsection SMTopology_Simplicity_Test_Primitive Detect Repetition
+It can be proven that if the canonical form can be expressed as concatenation of two or more copies of the same path, the path can never be made to be intersection free. So the first step of the algorithm is to detect repetition.
```

Comment:
```suggestion
It can be proven that if the canonical form can be expressed as concatenation of two or more copies of the same path, the path can never be made intersection free. So the first step of the algorithm is to detect repetition.
```